src/main/scala/com/github/sbt/git/GitPlugin.scala

```diff
@@ -21,6 +22,7 @@ object SbtGit {
     val gitUncommittedChanges = SettingKey[Boolean]("git-uncommitted-changes", "Whether there are uncommitted changes.")
 
     // A Mechanism to run Git directly.
+    @cacheLevel(include = Array.empty)
```

Comment:
Would it be an option to use @transient annotation which would be less boilerplate

Reply:
Good idea - https://github.com/sbt/sbt/issues/8204